src/I2SClocklessLedDriver.h

```diff
@@ -182,16 +189,9 @@ enum displayMode
     LOOP,
     LOOP_INTERUPT,
 };
-
+/*
 int MOD(int a, int b)
```

Comment:
I already had a MOD that was then causing errors for the linker. Is there a better way to handle this potential clash?